client/utils.go

```diff
@@ -0,0 +1,38 @@
+package client
+
+import (
+	"github.com/binance-chain/double-sign-evidence/types/bsc"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
```

Comment:
as we discussed, shall we remove the direct dependence on go-ethereum?

Reply:
This repo has LGPL license. Direct dependence on go-ethereum is ok.